src/status_im/contexts/wallet/wallet_connect/responding_events.cljs

```diff
@@ -132,8 +134,20 @@
                                           :event                :wallet-connect/send-response
                                           :wallet-connect-event event}))
                :on-success  (fn []
+                              (when platform/android?
+                                (rf/dispatch [:wallet-connect/redirect-to-dapp]))
```

Comment:
did you try it on iOS as well? I wonder what happens when we try to open the dapp.

Reply:
I first try it with `url` and it was working fine in both platform but with `native` as i couldn't test it with installed app still don't know. 
I may create a simple dapp to test logic if it doesn't take time

Reply:
> I may create a simple dapp to test logic if it doesn't take time

This may be too time consuming I guess

Reply:
yeah i thought too, another option is to build app in release mode and test it with https://explorer.walletconnect.com/status, will let you know if it works fine

Reply:
I removed the platform condition to have chance to test it with both platform, if in iOS it cause any issue i will disable it, in wallet connect doc it didn't mention any issue about iOS, lets see